pkg/epinio/utils/download.js

```diff
@@ -0,0 +1,18 @@
+import JSZip from 'jszip';
+
+/**
+ * Custom function since we need to append the filetype, we cannot use the one from the exported '@shell/utils/download'
+ * @param {*} files {[fileName1]:data1, [fileName2]:data2}
+ * @returns
+ */
+export async function generateZip(files) {
+  const zip = new JSZip();
+
+  for ( const fileName in files) {
+    zip.file(fileName === 'values' ? `${ fileName }.yml` : `${ fileName }.gz`, files[fileName]);
```

Comment:
given we do stuff specific to file contents and we only use this in a single place it shouldn't be a generic function. it can just be done inline where it's needed